templates/travel/components/activities/Activities.js

```diff
@@ -0,0 +1,61 @@
+import ActiveSvg from '../svg/ActiveSvg';
+import ArtisticSvg from '../svg/ArtisticSvg';
+import DrinksSvg from '../svg/DrinksSvg';
+import FashionSvg from '../svg/FashionSvg';
+import FoodSvg from '../svg/FoodSvg';
+import MusicSvg from '../svg/MusicSvg';
+import NatureSvg from '../svg/NatureSvg';
+import NightlifeSvg from '../svg/NightlifeSvg';
+import ToursSvg from '../svg/ToursSvg';
+import WaterSvg from '../svg/WaterSvg';
+
+export default function Activities(props) {
+  return (
+    <section className='travel-activities pb4 pt3 relative'>
+      <div className='max-width-3 mx-auto px1 md-px2'>
+        <h3 className='bold h1 line-height-2 mb2 md-hide lg-hide' aria-hidden='true'>
+          Browse
+          <br />
+          by activity
+        </h3>
+        <h3 className='bold h1 line-height-2 mb2 xs-hide sm-hide center'>Browse by activity</h3>
+      </div>
+      <div className='overflow-scroll'>
+        <div className='travel-overflow-container'>
+          <div className='flex justify-center p1 md-px1 mxn1'>
+            {[
```

Comment:
Some I converted from result of converter and then reverse engineered the source template. Next one I think should port the Mustache files rather than the output of the conversion. There are few like this around the place I have not done yet.